modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -335,7 +335,7 @@
                   "202 without a transactionId": {
                     "value": {
                       "data": {
-                        "id": "c21ca745-8f00-4222-af2c-6e1fb2216813",
+                        "id": "5226f34c-0a8a-47f3-ac38-4cfcba5541c4",
```

Comment:
Not a blocker for merging, but let's normalize these IDs & dates so that we don't have to resubmit their changes every time we regenerate the docs. This'll simplify the diffs so that only the parts that _meaningfully_ change will be present.

Banana Peels did this by [normalizing the responses](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/support/doc_helpers.rb#L5-L41) so that the IDs and dates were constant.

Reply:
@cilestin [Here](https://jira.devops.va.gov/browse/API-44648) is a new ticket to address normalizing the responses.